relay-general/src/store/transactions/processor.rs

```diff
@@ -2582,4 +2594,11 @@ mod tests {
         "db.sql.query",
         ""
     );
+
+    span_description_test!(
+        span_description_scrub_cache,
+        "abc:12:{def}:{34}:{fg56}:EAB38:zookeeper",
+        "cache.get_item",
+        "abc:*:*:*:*:*:zookeeper"
+    );
```

Comment:
Would be great to have a negative test as well. That is, `op = cache...` but a none-matching description.

Reply:
+1